source/robot_model/include/robot_model/Model.hpp

```diff
@@ -73,19 +73,24 @@ class Model {
                                                        unsigned int frame_id);
   
   /**
-   * @brief It checks if the vector's elements are inside at their limits
+   * @brief Check if the vector's elements are inside the parameter limits
```

Comment:
This might just be me but is this ok or should we change `vector's elements` to `elements of the vector` @eeberhard ?

Reply:
As the resident linguist, I can say that both options are fine in this situation. The elements do indeed belong to the vector, so the possessive case `vector's elements` is valid. It's a bit more compact than the genitive composite case `elements of the vector` and has the same meaning.